plugins/rbac-backend/docs/permissions.md

```diff
@@ -126,3 +126,9 @@ Resource type permissions on the other hand are basic named permissions with a r
 | ------------------ | ------------- | ------ | ----------------------------------------------------------------------------------------------------------- | ------------------- |
 | topology.view.read |               | read   | Allows the user to view the topology plugin                                                                 | X                   |
 | kubernetes.proxy   |               |        | Allows the user to access the proxy endpoint (ability to read pod logs and events within Showcase and RHDH) | catalog.entity.read |
+
+## Bulk Import
+
+| Name        | Resource Type | Policy | Description                                                                                                                                                                          | Requirements |
+| ----------- | ------------- | ------ | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ | ------------ |
+| bulk.import | bulk-import   |        | Allows the user to access the bulk import endpoints (listing all repositories and organizations accessible by all GitHub integrations, as well as managing the import requests, ...) | X            |
```

Comment:
I'm adding the following to the app-config to fetch the permission for the bulk import plugin 

```
permission:
  enabled: true
  rbac:
    policies-csv-file: /Users/dsantra/Documents/backstage-plugins/rbac-policy.csv
    policyFileReload: true
    pluginsWithPermission:
      # - kubernetes
      - bulk-import  # also tried with bulk.import
      - catalog
      - scaffolder
      # - policy
    admin:
      superUsers:
        - name: user:default/debsmita1
      users:
        # - name: user:default/debsmita1
        - name: group:default/admins
  
```
    


but I do not see the bulk import permission listed in the role form
![Screenshot 2024-08-13 at 11 31 42 AM](https://github.com/user-attachments/assets/ad2db7b2-a97b-4c61-9726-0840a5c26d66)

Reply:
@debsmita1 I was able to get it to show up in the UI but I had to set `bulk-import-backend` instead of `bulk-import` under the `pluginsWithPermission` section.
![Screenshot from 2024-08-16 08-46-18](https://github.com/user-attachments/assets/eca9e8d3-4dbe-498e-8244-ba216bcd381a)
Looks like the [pluginId](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/bulk-import-backend/src/plugin.ts#L32) for the backend plugin is `bulk-import-backend`. I wonder if it would make more sense to update it to `bulk-import` to match the other plugins. What do you think? @debsmita1 @rm3l

Reply:
+1 for `bulk-import` as the plugin id

Reply:
Sure, I'll update it to `bulk-import` in this PR. Thanks for catching this.

Reply:
[`4704322` (#1890)](https://github.com/janus-idp/backstage-plugins/pull/1890/commits/47043220084a6e77e9023f19e36deb55fb2f790b) renames the `pluginId` to `bulk-import`. PTAL.
Note that this caused the bulk import backend API base path to be changed accordingly, from `/api/bulk-import-backend` to `/api/bulk-import`.